.circleci/config.yml

```diff
@@ -456,7 +463,18 @@ jobs:
       - run:
           name: Ethereum Contract Unit Tests
           command: |
-            pytest $(circleci tests glob "tests/blockchain/eth/contracts/**/test_*.py" | circleci tests split --split-by=timings)
+            pytest $(circleci tests glob "tests/blockchain/eth/contracts/base/**/test_*.py" "tests/blockchain/eth/contracts/integration/**/test_*.py" "tests/blockchain/eth/contracts/lib/**/test_*.py" "tests/blockchain/eth/contracts/main/**/test_*.py" | circleci tests split --split-by=timings)
+      - capture_test_results
+
+  contracts_upgradeability:
+    <<: *python_37_base
+    parallelism: 1
+    steps:
+      - prepare_environment
+      - run:
+          name: Ethereum Contract Upgradeability Test
+          command: |
+            pytest $(circleci tests glob "tests/blockchain/eth/contracts/test_contracts_upgradeability.py" | circleci tests split --split-by=timings)
```

Comment:
After checking the performance of the CI build, I think having a separate job for this test is unnecessary. It blocks a CI box for some time, while if it were included in the `contracts` job, it will probably be executed without any overhead due to current parallelism.

Reply:
The main reason for extracting was readability of test results, like in 1459, where only one contract test failed, I like how it looks

Reply:
Right, I understand, but we have a limited number of CI boxes and using one of them for a single test that runs for 21 seconds doesn't seem a good idea to me.

Reply:
makes sense, I'll change it back

Reply:
✔️